Lab_15_Pig_Latin/functions.test.js

```diff
@@ -0,0 +1,25 @@
+const { translateToPigLatin } = require("./functions")
+
+describe("translateToPigLatin()", () => {
+    it(`takes in 'pickle' and returns 'icklepay'`, () => { // regular consonant
+        expect(translateToPigLatin("pickle")).toEqual("icklepay")
+    })
+    it(`takes in 'Mississippi' and returns 'ississippimay'`, () => { // capitalized word
+        expect(translateToPigLatin("Mississippi")).toEqual("ississippimay")
+    })
```

Comment:
```suggestion
    it(`works with capitalized words by lowercasing them`, () => {
        expect(translateToPigLatin("Mississippi")).toEqual("ississippimay")
    })
```

Reply:
My opinion about test names explained in this example: lines 7 and 8 originally are too similar to each other.
The `it` blocks should be more general, and you can have multiple assertions in that block:

```js
    it(`works with capitalized words by lowercasing them`, () => {
        expect(translateToPigLatin("Mississippi")).toEqual("ississippimay")
        expect(translateToPigLatin("Colorado")).toEqual("oloradocay")
        expect(translateToPigLatin("Georgia")).toEqual("eorgiagay")
    })
```